flink-runtime-web/web-dashboard/src/app/app.module.ts

```diff
@@ -83,7 +84,7 @@ export function AppInitServiceFactory(
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [BrowserModule, AppRoutingModule, NgZorroAntdModule, FormsModule, HttpClientModule, BrowserAnimationsModule],
+    imports: [BrowserModule, AppRoutingModule, NgZorroAntdModule, FormsModule, HttpClientModule, BrowserAnimationsModule, ShareModule],
```

Comment:
This seems to be improperly formatted, both the import and this line here. If the tooling doesn't complain then I guess we have no tooling for the UI for this, in which case feel free to disregard.

Reply:
I fixed the formatting, but no tooling complains here. Would be nice to use an auto formatter for the project, in particular for noobies like me.